charm-seeder/cli.ts

```diff
@@ -503,13 +574,17 @@ async function generateReport() {
   console.log(`Report generated: ${reportPath}`);
 }
 
-async function verifyCharm(id: string, prompt: string): Promise<string> {
+function verifyCharm(id: string, prompt: string): Promise<string> {
+  return verifyCharmWithPage(id, prompt, page);
+}
+
+async function verifyCharmWithPage(id: string, prompt: string, targetPage: Page): Promise<string> {
   // FIXME(ja): can we navigate without causing a page reload?
-  await page.goto(new URL(`/${name}/${id}`, toolshedUrl).toString());
-  addErrorListeners();
+  await targetPage.goto(new URL(`/${name}/${id}`, toolshedUrl).toString());
```

Comment:
perhaps we can fix this? clicking on the space name, then clicking on the charm id - rather than refreshing every single time

we could probably reduce the timeout below

perhaps also we could have iframe emit "loaded" event ? something like "common-iframe-ready" when the inner iframe is done loading (perhaps you and @jsantell can chat about this?)

with both of these the 5 second sleep can go away!

(although we would probably need to rework the error listener work - to not re-add but empty it out after fininshed loading `/space`)

Reply:
`<common-iframe-sandbox>` emits a `"load"` event, but does not bubble outside of shadow DOM, though could change. Loading of the iframe document may not necessarily be "loaded" from the perspective of all content in the iframe being initialized as well. May still have race conditions between "no visual representation" (listen to load event) and "load event fired"